test/integration/main.test.js

```diff
@@ -166,40 +166,37 @@ describe('history', () => {
 
         this.timeout(10000);
 
-        let chatHistory = new ChatEngine.Chat('chat-history-2', false);
-
-        chatHistory.on('$.history.tester', (a) => {
-
-            assert.equal(a.event, 'tester');
+        chatHistory = new ChatEngine.Chat('chat-history-8', false);
 
-            count += 1;
+        // let i = 0;
```

Comment:
Is there a reason to keep these comment blocks around?

Reply:
That's a good question. I feel like I have history set to expire in 1 day or something on this key. let me undo that.